vmdb/app/models/mixins/miq_web_server_worker_mixin.rb

```diff
@@ -54,7 +54,7 @@ def self.build_command_line(*params)
     end
 
     def self.all_ports_in_use
-      self.server_scope.all.collect { |w| w.port unless w.is_stopped? && !MiqProcess.is_worker?(w.pid)}.compact
+      server_scope.select { |w| MiqProcess.is_worker?(w.pid) || !w.is_stopped? }.collect(&:port)
```

Comment:
@matthewd We might want to flip the `||` to `!w.is_stopped? || MiqProcess.is_worker?(w.pid)`, only because the `MiqProcess.is_worker?` goes out to the system to determine if that particular pid is a worker process, which is more expecting that checking an attribute.  By flipping, we can at least leverage some of the short circuiting so we don't go out to the system.  Thoughts?